llvm/lib/CodeGen/GlobalISel/IRTranslator.cpp

```diff
@@ -2474,6 +2475,49 @@ bool IRTranslator::translateKnownIntrinsic(const CallInst &CI, Intrinsic::ID ID,
 
     return true;
   }
+
+  case Intrinsic::experimental_vector_interleave2: {
+    Value *Src0 = CI.getOperand(0);
+    Value *Src1 = CI.getOperand(1);
```

Comment:
Can move these definitions below and early exit based on `OpType.isFixedVector()`

Reply:
Done.